routes/originPod.js

```diff
@@ -6,25 +6,33 @@ const router = require('express').Router()
 const requestPromise = require('request-promise-native')
 const { uniq } = require('lodash')
 
-// TODO: where does this come from? GitHub??
 router.get(
   '/:name/revisions',
   asyncMiddleware(async (request, response) => {
     const { name } = request.params
-    const url = `https://cocoapods.org/api/v1/pods/${name}`
+    const url = `https://trunk.cocoapods.org/api/v1/pods/${name}`
     const answer = await requestPromise({ url, method: 'GET', json: true })
-    return response.status(200).send(uniq(answer.versions.map(x => x.num)))
+    return response.status(200).send(uniq(answer.versions.map(x => x.name)))
   })
 )
 
-// TODO: where does this come from? GitHub??
 router.get(
   '/:name',
   asyncMiddleware(async (request, response) => {
     const { name } = request.params
-    const url = `https://cocoapods.org/api/v1/pods?per_page=100&q=${name}`
-    const answer = await requestPromise({ url, method: 'GET', json: true })
-    const result = answer.crates.map(x => {
+    const algolia = {
+        appID: 'WBHHAMHYNM',
+        apiKey: '4f7544ca8701f9bf2a4e55daff1b09e9'
```

Comment:
Should this key be fetched from the configuration?

Reply:
Good question, this API key is managed by the CocoaPods admins and it's not something we can generate/replace. I took it from [here](https://github.com/CocoaPods/cocoapods.org/blob/bcd28fe68cd82390e0f66c9988b9e109eed58695/assets/javascripts/search.config.js#L689).